src/kontrol/deployment.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, name: str, accounts: dict | None = None) -> None:
 
     def generate_header(self) -> list[str]:
         lines = []
+        lines.append(f'// This file was autogenerated by running `kontrol summary`. Do not edit this file manually.\n')
```

Comment:
The code quality CI jobs complains about an `f`-string not having placeholders, so perhaps it should be:
```suggestion
        lines.append('// This file was autogenerated by running `kontrol summary`. Do not edit this file manually.\n')
```

Reply:
Yes, I could run make now and also noticed that. It's fixed now.